src/site/paragraphs/alert.drupal.liquid

```diff
@@ -0,0 +1,88 @@
+    "entity": {
+    "fieldAlertTitle": "You'll need to go to eBenefits to authorize us to share your health information through the Veterans Health Information Exchange.",
+    "fieldHelpText": null,
+    "fieldAlertType": "information",
+    "fieldReusability": "reusable",
+    "fieldAlertContent": {
+    "entity": {
+    "fieldWysiwyg": {
+    "processed": "<p>To use this feature, you'll need a Premium <strong>DS Logon</strong> account. Your My HealtheVet or ID.me credentials won’t work on the eBenefits website. Go to eBenefits to sign in, register, or upgrade your <strong>DS Logon</strong> account to Premium.<br /><a class=\"usa-button-primary\" href=\"https://www.ebenefits.va.gov/ebenefits/vapii\">Go to eBenefits</a>\n</p>"
+    }
+    }
+    }
+    }
+    },
+    "fieldVaParagraphs": []
+    }
+{% endcomment %}
+{% assign alertType = entity.fieldAlertType %}
+{% if alertType = "information" %}
+    {% assign alertType = "info" %}
+{% endif %}
+
+{% if entity.fieldAlertBlockReference != empty %}
+    {% assign alertBlock = entity.fieldAlertBlockReference.entity %}
+    {% if alertBlock.fieldAlertContent %}
+        {% assign expander = alertBlock.fieldAlertContent.entity %}
+    {% endif %}
+
+    <div class="usa-alert usa-alert-{{ alertType }}" role="alert">
+        <div class="usa-alert-body">
+            <h3 class="usa-alert-heading">
+                {{ alertBlock.fieldAlertTitle }}
+            </h3>
+            {% if alertBlock.fieldHelpText %}
+                <p>{{ alertBlock.fieldHelpText }}</p>
+            {% endif %}
+            {% if expander.fieldTextExpander %}
+
+                {% comment %}
+                    NOTE: .additional-info-container is a class utilized by
+                    createAdditionalInfoWidget.js to add toggle functionality to info alerts
+                {% endcomment %}
+
+                {% if alertBlock.fieldAlertTitle contains 'homeless' %}
+                    {% assign analyticsId = 'nav-crisis-homelessness-expander' %}
+                    {% elsif expander.fieldTextExpander contains 'support' %}
+                    {% assign analyticsId = 'nav-crisis-get-support-247' %}
+                    {% elsif expander.fieldTextExpander contains 'care now' %}
+                    {% assign analyticsId = 'nav-crisis-get-care-now' %}
+                {% else %}
+                    {% comment %}
+                        Nothing required!
+                    {% endcomment %}
+                {% endif %}
+
+                <div data-analytics="{{ analyticsId }}" class="form-expanding-group borderless-alert additional-info-container">
+                    <div class="additional-info-title">{{ expander.fieldTextExpander }}</div>
+
+                    {% if expander.fieldWysiwyg %}
+                        <div class="additional-info-content usa-alert-text" hidden>{{ expander.fieldWysiwyg.processed }}</div>
+                    {% endif %}
+                </div>
+            {% endif %}
```

Comment:
It seems like we could possibly include `src/site/blocks/alert.drupal.liquid` in the if-block instead of adding this HTML? Any way we can share code I think would be great.